packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -100,14 +101,18 @@ class PopupMenuDivider extends PopupMenuEntry<Never> {
   /// Creates a horizontal divider for a popup menu.
   ///
   /// By default, the divider has a height of 16 logical pixels.
-  const PopupMenuDivider({ Key? key, this.height = _kMenuDividerHeight }) : super(key: key);
+  const PopupMenuDivider({ Key key, this.height = _kMenuDividerHeight, this.color = _kMenuDividerColor })
+      : super(key: key);
 
   /// The height of the divider entry.
   ///
   /// Defaults to 16 pixels.
   @override
   final double height;
 
+  @override
+  final Color color;
```

Comment:
This is a public API it requires documentation

Reply:
@pedromassangocode i should add documentation for that?

Reply:
Yes